src/App.js

```diff
@@ -1,26 +1,88 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import {BrowserRouter as Router, Link, Route, Switch} from "react-router-dom";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div className="App">
+        <header className="App-header">
+          <a
+              className="App-link"
+              href="/games"
+              // target="_blank"
+              rel="noopener noreferrer"
+          >
+            Arobs Games
+          </a>
+          <Router>
```

Comment:
Try to separate this logic in 2 components, one for the Routes and One for the links.